Read and store ST configuration in yaml (#9630)
strimzi/strimzi-kafka-operator: development-docs/TESTING.md

System tests can be configured by several environment variables, which are loaded before test execution.

Variables can be defined via environmental variables or a configuration file; this file can be located anywhere on the file system as long as a path is provided to this file.
The path is defined by the environmental variable `ST_CONFIG_PATH`; if the `ST_CONFIG_PATH` environmental variable is not defined, the default config file location is used `systemtest/config.yaml`.
Loading of system configuration has the following priority order:
1. Environment variable
2. Variable defined in the configuration file
3. Default value

> After every systemtest tes run all env variables is automatically stored into $TEST_LOG_DIR/test-run.../config.yaml so it test run can be easily reproduced just by running

Reviewer: ```suggestion
After every systemtest test run all env variables are automatically stored into $TEST_LOG_DIR/test-run.../config.yaml so each test run can be easily reproduced just by running
```
